renku_notebooks/api/notebooks.py

```diff
@@ -94,13 +99,19 @@ def user_servers(user, **query_params):
         - servers
     """
     servers = [
-        UserServer.from_js(user, js, config.k8s.client)
-        for js in config.k8s.client.list_servers(user.safe_username)
+        UserServerManifest(i)
+        for i in config.k8s.client.list_servers(user.safe_username)
```

Comment:
`i` is convention for loop index (int), we should use something else since it's a server (`s` at the least)